### ISSUE TITLE
Maybe a bug in the Examples File: AnyDevice

### ISSUE THREAD

**Issue by huddx01** [ztalbot2000/homebridge-cmd4#113]
**Describe The Bug:**

State of a Security System is not written correctly when polling

**To Reproduce:**

I think it should be 

```
     case "SecuritySystemTargetState":
         {
            writeData( device, characteristic, option );

            // Set to done
            writeData( device,"SecuritySystemCurrentState", option );

            break;
         }
```

instead of...

     case "SecuritySystemTargetState":
         {
            writeData( device, characteristic, option );

            // Set to done
            writeData( device,"SecuritySystemAlarmState", option );

            break;
         }

**Comment by ztalbot2000** [comment]
Hi,

   Thanks for your bug report.  I'm trying to remember the details around
this characteristic.  You are probably correct, but the difference may be
on what the device is.  I'm trying to remember if there are two different
devices that could use this characteristic and each one differently.

I'll have to get back to you.

Ttyl,
John

**Comment by huddx01** [comment]
Hi. 

I have tried to implement a securitysystem “dummy” device… which responds correctly. 

So I tried to choose the AnyDevice script, which gets/sets the corresponding values….

Thank you so far…

**Comment by ztalbot2000** [comment]
Hi,

  Sorry, I did not forget about you, but from the bug description I did not
put it at the top of my list.  I am working on the Schema portion for Cmd4
so that you don't need to edit the config.json manually. As you can
imagine, it's highly involved.
I'll stop and look at yours now.

Ttyl,
John

**Comment by ztalbot2000** [comment]
Hi I just released v5.1.1 fixing AnyDevice.  I also created and tested the new example script SecuritySystem.js

The comments in the script has the config.json that I used as well.  There is interesting information I also note in the script.The securitySystemTargetState does not align with the CurrentState.  To ARM/DISARM you must run the script from the command line.

if you have further issues, just continue writing to this ticket, even though I will now close it.

take care,
John
